flake.nix

```diff
@@ -140,7 +140,7 @@
             tag = "latest";
             # breaks binary reproducibility, but makes usage easier
             created = "now";
-            contents = with pkgs; [ hoprd iana-etc cacert ];
+            contents = with pkgs; [ hoprd iana-etc cacert bash ];
```

Comment:
@ausias-armesto pls use this generated image... to see whether you are not missing anything else first.

Reply:
^^